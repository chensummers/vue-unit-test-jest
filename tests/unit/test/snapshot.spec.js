import YesNoComponent from "@/components/YesNoComponent.vue";
import { mount } from '@vue/test-utils'


describe('YesNoComponent snapshot', () => {
    test('renders correctly', () => {
        const wrapper = mount(YesNoComponent)
        expect(wrapper.element).toMatchSnapshot()
    })
})