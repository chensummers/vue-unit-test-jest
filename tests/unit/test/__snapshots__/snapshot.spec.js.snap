// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YesNoComponent snapshot renders correctly 1`] = `
<div>
  <button
    class="yes"
  >
    Yes
  </button>
   
  <button
    class="no"
  >
    No
  </button>
</div>
`;
